pilot/pkg/model/push_context.go

```diff
@@ -102,6 +102,9 @@ type PushContext struct {
 	// AuthNPolicies contains a map of hostname and port to authentication policy
 	AuthnPolicies processedAuthnPolicies `json:"-"`
 
+	// (beta) Authn policies for each namespace.
```

Comment:
doc string starts with type name.

Reply:
also suggest to rename to `xxByNamespace` for the consistency of rest fields.

Or make the type AuthnBetaPolicies carrying this info.

Reply:
Done